src/components/RoadmapList/RoadmapList.tsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react';
+import ReactPaginate from 'react-paginate';
+import { Link } from 'react-router-dom';
+
+import { getRoadmapList } from 'src/api';
+
+import Button from 'components/ui/Button';
+import Container from 'components/ui/Container';
+import Icon from 'components/ui/Icon';
+
+import { RoadmapListItem } from './type';
+
+import style from './RoadmapList.module.css';
+
+const RoadmapList = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPage, setTotalPage] = useState(1);
+  const [list, setList] = useState<RoadmapListItem[]>([]);
+
+  const pageSize = 6;
+
```

Comment:
Better put this const out of component and write like const with Uppercase, and make other name because its not page size. It's roadmaps quantity on page